app/views/admin/editions/_edition_view_edit_buttons.html.erb

```diff
@@ -37,7 +37,8 @@
         </div>
         <div id="collapseOne" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne">
           <div class="panel-body">
-            <p>Send the preview link to someone so they can see a preview of how the document will appear on GOV.UK. No password or GOV.UK Signon account is needed.</p>
+            <p>Send this preview link to someone so they can see the content and how the document will appear on GOV.UK.</p>
+            <p>No password is needed and anyone with the preview link can view it. You're responsible for who you share draft documents with. </p>
```

Comment:
There's an extra space but the content lgtm
```
<p>No password is needed and anyone with the preview link can view it. You're responsible for who you share draft documents with.</p>
```

Reply:
@davidgisbey thanks, I've taken it out and just pushed it up

Reply:
lgtm! :shipit: